template/app/src/main/AndroidManifest.xml

```diff
@@ -3,6 +3,8 @@
     package="co.nimblehq.template">
 
     <uses-permission android:name="android.permission.INTERNET" />
+    <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION" />
+    <uses-permission android:name="android.permission.CAMERA" />
```

Comment:
This feels like sample code, would you agree? 💭

Reply:
Good point 👍🏻 Shall we also update `HomeViewModel.requestPermissions()` to emit an empty array instead? Or remove that method entirely? 🤔

Reply:
I'm fine to keep `HomeViewModel.requestPermissions()` as it is (for the `samples`) 💪

Reply:
If thats the case then running the project after generating could lead to a crash because those permissions weren't declared in the manifest 🤔

Reply:
Can we do this?

- `template`: Only define the dependency, nothing else
- `samples`: Define dependency, declare permissions, integrate home-fragment & home-view model

Please let me know if I got the wrong idea here 🙇

Reply:
That could definitely work 👀 Will apply it like that 😄